lib/lobbyist.rb

```diff
@@ -165,14 +164,12 @@
 require 'lobbyist/v2/review_flag'
```

Comment:
@sreevs 
In https://github.com/customerlobby/lobbyist-ruby/pull/128/files, `review_flag` was added. (https://github.com/customerlobby/lobbyist-ruby/pull/128/files#diff-f01047f9ebeabab935880606888e5a54R164)

It has not been removed in this PR. We should remove this.